Detections/PulseConnectSecure/PulseConnectSecureVPN-PasswordSpray.yaml

```diff
@@ -1,7 +1,8 @@
 id: 1fa1528e-f746-4794-8a41-14827f4cb798
-name: Large Number of Distinct Failed User Logins
+name: Potential Password Spray Attempts
```

Comment:
I would be careful with this rename, we don't know what Pulse Secure is identifying here.  We should add - `PulseConnectSecure - ` to the beginning though.

Reply:
Thanks Shain. Renamed it and also added PulseConnectSecure in the begining of the name to identify Data Source.